example/web/index.dart

```
@@ -76,47 +76,47 @@ void setupAssists() {
 }
 
 void setupCompile() {
-  final editor = createEditor(querySelector('#compileSection-v2 .editor'));
+  final editor = createEditor(querySelector('#compileSection-v2 .editor')!);
   final output = querySelector('#compileSection-v2 .output');
   final button = querySelector('#compileSection-v2 button') as ButtonElement;
   button.onClick.listen((e) {
-    final source = editor.getDoc().getValue();
+    final source = editor.getDoc()!.getValue();
     final compile = {'source': source};
     final sw = Stopwatch()..start();
     post(
       '$_uriBase/dartservices/v2/compile',
       body: json.encode(compile),
-    ).then((response) => output.text = '${_formatTiming(sw)}${response.body}');
+    ).then((response) => output!.text = '${_formatTiming(sw)}${response.body}');
```

just a style note: you might move this `!` to the end of line 80, so that `output` is definitely an element, and we fail faster.

---

done